src/util.ts

```diff
@@ -92,7 +92,11 @@ export async function convertToMenuObj(text: string) {
 }
 
 export function getRandomGifLink(type: string): string {
-  const theme = type.length > 0 ? type.toLocaleLowerCase() : 'notheme';
+  let theme = type.length > 0 ? type.toLocaleLowerCase() : 'notheme';
+  if (theme === ':broccoli:') {
+    theme = 'vegan';
```

Comment:
Might include e.g. diary and eggs as well.
```suggestion
    theme = 'vegetarian';
```

Also, gifs?